CMakeLists.txt

```diff
@@ -195,66 +186,66 @@ if(POLYFEM_THREADING STREQUAL "CPP")
     target_compile_definitions(polyfem PUBLIC _GLIBCXX_USE_TBB_PAR_BACKEND=0)
 elseif(POLYFEM_THREADING STREQUAL "TBB")
     include(onetbb)
-    polyfem_target_link_system_libraries(polyfem PUBLIC TBB::tbb)
+    target_link_libraries(polyfem PUBLIC TBB::tbb)
     target_compile_definitions(polyfem PUBLIC -DPOLYFEM_WITH_TBB)
 endif()
 
 # libigl
 include(libigl)
-polyfem_target_link_system_libraries(polyfem PUBLIC igl::core)
+target_link_libraries(polyfem PUBLIC igl::core)
 
 if(LIBIGL_WITH_TRIANGLE)
-    polyfem_target_link_system_libraries(polyfem PUBLIC igl::triangle)
+    target_link_libraries(polyfem PUBLIC igl::triangle)
     target_compile_definitions(polyfem PUBLIC -DPOLYFEM_WITH_TRIANGLE)
 endif()
 
 # BVH
 include(BVH)
-polyfem_target_link_system_libraries(polyfem PUBLIC BVH_lib)
+target_link_libraries(polyfem PUBLIC BVH_lib)
 
 # MSHIO
 include(mshio)
-polyfem_target_link_system_libraries(polyfem PUBLIC mshio)
+target_link_libraries(polyfem PUBLIC mshio)
 
 # Geogram
 include(geogram)
-polyfem_target_link_system_libraries(polyfem PUBLIC geogram)
+target_link_libraries(polyfem PUBLIC geogram)
```

Comment:
why remove the other target that suppressed warnings?

Reply:
CPM does it for us automatically!